src/main/kotlin/com/wafflestudio/toyproject/team4/core/item/api/request/PostItemInquiryRequest.kt

```diff
@@ -13,27 +12,25 @@ data class PostItemInquiryRequest(
     val isSecret: Boolean,
     val title: String?,
     val content: String?,
-    val images: List<String>? = mutableListOf()
+    val images: List<String>?
 ) {
     init {
         if (content == null) throw CustomHttp400("문의 내용을 입력해주세요.")
         if (title == null) throw CustomHttp400("제목은 필수 입력값입니다.")
     }
 
-    fun toEntity(user: UserEntity, item: ItemEntity): InquiryEntity {
-        val newInquiry = InquiryEntity(
+    fun toEntity(user: UserEntity, item: ItemEntity) =
+        InquiryEntity(
             user = user,
             item = item,
             title = title!!,
             content = content!!,
```

Comment:
`title`과 `content`를 non-nullable로 수정 후에 이 부분도
```suggestion
            title = title,
            content = content,
```
로 수정 부탁드립니다.